### PR TITLE
tests: Add docker test to check environment.

### PR DESCRIPTION
**Pull request by amshinde**
Signed-off-by: Archana Shinde <archana.m.shinde@intel.com>

### CODE REVIEW

**Comment by devimc** [comment]
lgtm

File: tests/integration/docker/env.bats [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+setup() {
+	source $SRC/test-common.bash
+	clean_docker_ps
+	runtime_docker
+}
+
+@test "Verify LANG is not set in env" {
+	skip "this is not working (see https://github.com/01org/cc-oci-runtime/issues/324)"
+	if $DOCKER_EXE run -i ubuntu env | grep LANG; then false; else true; fi
```
Comment (dlespiau) [inline tests/integration/docker/env.bats]: ah, you don't seem to know about grep -v :)

```
$ echo ENV | grep ENV >/dev/null; echo $?
0
$ echo ENV | grep -v ENV >/dev/null; echo $?
1
```

**Comment by dlespiau** [comment]
lgtm

File: tests/integration/docker/env.bats (hunk 1 above)
Comment (amshinde) [inline tests/integration/docker/env.bats]: Just a copy from a previous test there :)

**Comment by chavafg** [comment]
qa-failed

**Comment by grahamwhaley** [comment]
In the qa, the 'exec' test *immediately after* this one failed - which I thus suspect may mean this test had some dangling effect that rolled onto the next one - so, @amshinde , can we look into that?

**Comment by chavafg** [comment]
retested, got 1 failure wrt #593, everything else looks ok

**Comment by chavafg** [comment]
qa-passed

**Pull request merged**